test/formal/opcodes.py

```diff
@@ -56,3 +56,6 @@ def SHR(x, y):
 
 def SAR(x, y):
 	return y >> x
+
+def BYTE(i, x):
+	return (x >> (256 - 8 * (i + 1))) & 0xff
```

Comment:
```suggestion
	return (x >> (x.size() - 8 * (i + 1))) & 0xff
```
Better to keep the opcodes agnostic to the bit sizes, so that we can prove stuff in lower precision, if necessary.